mousestyles/data/__init__.py

```diff
@@ -62,3 +62,98 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: str
+             where feature is one of ["AS", "F", "IS", "M_AS", "M_IS", "W"]
+    Returns
+    -------
+    output type: DataFrame
+    description: All the data from the feature directory into one data frame
+                 columns are strain, mouse, day, start, stop
+    """
+    # get all file names
+    # file_names = _os.listdir("intervals/" + feature)
+    file_names = _os.listdir(_os.path.join(data_dir, "intervals", feature))
+    file_names = [name for name in file_names if len(name) > 10]
+    # initialized data frame
+    # use for loop to load every file and concat to overall data frame
+    dt = pd.DataFrame()
+    for item in file_names:
+        strain = int(item.split("strain")[1].split("_mouse")[0])
+        mouse = int(item.split("mouse")[1].split("_day")[0])
+        day = int(item.split("day")[1].split(".npy")[0])
+        # path = "intervals/" + feature + "/" + item
+        path = _os.path.join(data_dir, "intervals", feature, item)
+        sub = np.load(path)
+        dt_sub = pd.DataFrame()
+        dt_sub["strain"] = [strain] * sub.shape[0]
+        dt_sub["mouse"] = [mouse] * sub.shape[0]
+        dt_sub["day"] = [day] * sub.shape[0]
+        dt_sub["start"] = sub[:, 0]
+        dt_sub["stop"] = sub[:, 1]
+        dt = pd.concat([dt, dt_sub])
+    # sort based on strain, mouse and day
+    dt = dt.sort(["strain", "mouse", "day"])
+    dt.index = range(dt.shape[0])
+    return dt
+
+
+def load_movement(strain, mouse, day):
+    """
+    Parameters
+    ----------
+    strain: int
+            nonnegative integer indicating the strain number
+    mouse:  int
+            nonnegative integer indicating the mouse number
+    day:    int
+            nonnegative integer indicating the day number
+    Returns
+    -------
+    output type: DataFrame
+    description: CT, CX, CY coordinates of the specified strain-mouse-day
+                 columns are t, x, y and isHB
+                 isHB indicates whether the point is in the home base or not
+    """
+    # load all four files of HB, CT, CX and CY data
+    # HB = np.load("txy_coords/C_idx_HB/C_idx_HB_strain" + str(strain) +
+    #     "_mouse" + str(mouse) + "_day" + str(day) + ".npy")
+    # CT = np.load("txy_coords/CT/CT_strain" + str(strain) +
+    #     "_mouse" + str(mouse) + "_day" + str(day) + ".npy")
+    # CX = np.load("txy_coords/CX/CX_strain" + str(strain) +
+    #     "_mouse" + str(mouse) + "_day" + str(day) + ".npy")
+    # CY = np.load("txy_coords/CY/CY_strain" + str(strain) +
+    #     "_mouse" + str(mouse) + "_day" + str(day) + ".npy")
+    HB = np.load(
+        _os.path.join(
+            data_dir,
+            "txy_coords/C_idx_HB/C_idx_HB_strain" + str(strain) +
```

Comment:
You might consider changing these to use [format strings](https://docs.python.org/3.5/library/string.html#formatstrings), which are in my opinion more readable.